modules/microshift-nw-ipv6-concept.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-intro-ipv6_{context}"]
+= IPv6 networking with {microshift-short}
+
+The {microshift-short} service defaults to IPv4 address families. However, IPv6 single-stack and IPv4/IPv6 dual-stack networking is available on supported platforms. When you set the values for IPv6 in the {microshift-short} configuration file and restart the service, settings managed by the OVN-Kubernetes network plugin are updated automatically. After migrating to dual-stack networking, both new and existing pods have dual-stack networking enabled.
+
+[IMPORTANT]
+====
+Plan for IPv6 before starting {microshift-short} for the first time because converting a cluster to and from single and dual networking stacks is not supported. Each {microshift-service} cluster network and service network allows only one value and the value cannot be changed after the service starts for the first time.
+====
+//Are we supporting migration? I see both in the Jiras
```

Comment:
We do, that is part of the dual stack implementation. This will be included in the ipv6 doc once all the PRs are merged (they are already open).